api/nakadi-event-bus-api.yaml

```diff
@@ -2328,6 +2386,61 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+  EventTypeAuthorization:
+    type: object
+    description: |
+      Authorization section for an event type. This section defines three access control lists: one for producing
+      events ('write'), one for consuming events ('read'), and one for administering an event type ('admin').
+      Regardless of the values of the authorization properties, administrator accounts will always be authorized.
+    properties:
+      admin:
+        type: array
+        description: |
+          An array of subject attributes that are required for updating the event type. Any one of the attributes
+          defined in this array is sufficient to be authorized. The catch-all item takes precedence over all others,
```

Comment:
I see 2 other options:
- an object whose fields are all optional, and each field represents a type of attribute. So you'd have for example:
```
{ "read":
  { "users": ["slava", "lionel", "*"],
    "teams": ["aruha", "flux"]
  }
}
```
I don't like it because it's hard to exhaustively define what the fields are, since types are implemented in the authorization plugin. But the wildcard could be anywhere.

The other option I see is to use a single array of strings, formatted as `<type>::<value>`:
```
{
"read": ["user::slava", "user::lionel", "*", "team::aruha", "team::flux"]
}
```
We'll have to split the strings, but otherwise it works well with wildcards. I would prefer this option over the one above.